akka-persistence-shared/src/test/scala/akka/persistence/journal/leveldb/SharedLeveldbJournalSpec.scala

```diff
@@ -90,7 +90,9 @@ class SharedLeveldbJournalSpec extends AkkaSpec(SharedLeveldbJournalSpec.config)
       val probeB = new TestProbe(systemB)
 
       val storeConfig = system.settings.config.getConfig("akka.persistence.journal.leveldb-shared")
-      system.actorOf(Props(classOf[SharedLeveldbStore], storeConfig), "store")
+      @nowarn
+      val sharedLeveldbStoreCls = classOf[SharedLeveldbStore]
+      system.actorOf(Props(sharedLeveldbStoreCls, storeConfig), "store")
```

Comment:
Here is also an interesting one. The class `SharedLeveldbStore` and gives a warning in 2.13.7 when passing it to `classOf`. We can't add `@nowarn` on method parameters, so I needed to extract it to a val.

Reply:
What was the warning?

Reply:
I wonder since it could mean also user code trying to create props with a class would end up with the same warning.